metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala

```diff
@@ -35,52 +39,76 @@ case class ScalafixProvider(
       scalaVersion: ScalaVersion,
       scalaBinaryVersion: ScalaBinaryVersion,
       classpath: JList[Path]
-  ): List[l.TextEdit] = {
+  ): Future[List[l.TextEdit]] = {
     val fileInput = file.toInput
     val unsavedFile = file.toInputFromBuffers(buffers)
```

Comment:
I know you didn't change this, but reading through this, there is a chance that is isn't unsaved, but rather the same thing as `fileInput`? If so, I think naming this `unsavedFile` is a bit misleading.

Reply:
Renamed it, you're right.